NAMESPACE

```diff
@@ -399,6 +399,7 @@ import(checkmate)
 import(ggplot2)
 import(methods)
 import(tibble)
+import(tidyverse)
```

Comment:
let's please avoid importing whole of `tidyverse`...

Reply:
I have removed all references to the metapackage `tidyverse`.  Instead, vignettes now
- Reference individual packages (eg `dplyr`) directly
- Use `requireNamespace()` to check that the required packages are installed and, if not, fail gracefully.